right-for-life_front/src/components/SiteContentContainer/setUseOrNotGoBack.js

```diff
@@ -0,0 +1,6 @@
+import { UPD_BACK_BTN_STATUS } from '../../rootConstants';
```

Comment:
If specific constants refer only to a specific component, it is better to save this constant in the "constants.js" file in the folder of a specific component

Reply:
Done